01 - Community/People/phibr0.md

```diff
@@ -32,6 +32,7 @@ publish: true
 - [[open-with|Open with]]
 - [[obsidian-relative-find|Relative Find]]
 - [[tabout|Tabout]]
+- [[Commander]]
```

Comment:
When I view this page in Obsidian with the new edit, the link `[[Commander]]` Showa up as a broken link.

I recommend making edits to the Hub in Obsidian, so that you can test them out as you are editing them.

The reason for the missing link is that the plugin links take the form `[[pluginID|plugin name]]` - so you would need to look up the plugin ID for Command for Commander and fix the new link please.